src/components/contribute/Contribute.tsx

```diff
@@ -285,9 +289,9 @@ const ContributeComponent: React.FunctionComponent = () => {
               {/* [Field] NSFW */}
               <div className="form-group">
                 <div className="form-row">
-                  <label className={cx('col-12', 'mb-2', errors.isNsfw && 'text-danger')}>
+                  <legend className={cx('col-12', 'mb-2', errors.isNsfw && 'text-danger')}>
```

Comment:
This change may seem a little counter-intuitive. The [`jsx-a11y/label-has-associated-control`](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md) rule requires that, for accessibility reasons, any `<label>` tag have a `for` attribute pointing to an HTML control. When using radio buttons, the `<label>` tags linked to the `<input>` controls are the labels for the options themselves. Therefore, the text for the prompt needs to use a different tag, and the preferred tag for this use case is `<legend>`.